items/models.py

```diff
@@ -1,17 +1,25 @@
+import datetime
+
+from django.core import validators
 from django.db import models
+from django.utils.translation import gettext_lazy as _
 
 from core.db.base import BaseAbstractModel
 
 from products.models import Product
 
 
 class Item(BaseAbstractModel):
-    product = models.ForeignKey(
-        Product, related_name='items', on_delete=models.CASCADE
-    )
+    product = models.ForeignKey(Product, related_name="items", on_delete=models.CASCADE)
     latest_delivery_date = models.DateField()
-    quantity = models.IntegerField()
-    valid_from = models.DateField(null=True, blank=True)
+    quantity = models.IntegerField(
+        validators=[
+            validators.MinValueValidator(
```

Comment:
Why not https://docs.djangoproject.com/en/3.0/ref/models/fields/#positiveintegerfield ?

Reply:
Why not. Fixed. Thanks.